problems/1.types+operators.js

```diff
@@ -2,17 +2,26 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+function isNumber(val) {
+  if(typeof val === "number") {
```

Comment:
Nice, can go ahead and just return condition since it already evaluates to true or false.